Description: Upstream changes introduced in version 2.0.0-10~nattyubuntu3
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 bumblebee (2.0.0-10~nattyubuntu3) natty; urgency=high
 .
   * Fix to issue #434..
 .
 The person named in the Author field signed this changelog entry.
Author: Martin Juhl (MrMEEE) <mj@casalogic.dk>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- bumblebee-2.0.0.orig/install-files/bumblebee-configuration
+++ bumblebee-2.0.0/install-files/bumblebee-configuration
@@ -294,26 +294,31 @@ please manually enter configuration in:\
 /usr/local/bin/bumblebee-enablecard and /usr/local/bin/bumblebee-disablecard\n\n\
 Some examples are available in /usr/share/doc/bumblebee/.\n\n\
 Press any key to continue..." 30 100
-fi
-fi
  if [ ! -f /usr/local/bin/bumblebee-enablecard ]; then
   # Not installed
-  cp /usr/share/doc/bumblebee/bumblebee-enablecard.template.ubuntu /usr/local/bin/bumblebee-enablecard
+  cp /usr/share/doc/bumblebee/bumblebee-enablecard.template /usr/local/bin/bumblebee-enablecard
  else
   # Already Exists
   echo
-  echo "nVidia card enable-script: /usr/local/bin/bumblebee-enablecard, already exists not overwriting."
+  echo "nVidia card enable-script: /usr/local/bin/bumblebee-enablecard exists moving old script to /usr/local/bin/bumblebee-enablecard.old."
   echo
+  mv /usr/local/bin/bumblebee-enablecard /usr/local/bin/bumblebee-enablecard.old
+  cp /usr/share/doc/bumblebee/bumblebee-enablecard.template /usr/local/bin/bumblebee-enablecard
  fi
  if [ ! -f /usr/local/bin/bumblebee-disablecard ]; then
   # Not installed
-  cp /usr/share/doc/bumblebee/bumblebee-disablecard.template.ubuntu /usr/local/bin/bumblebee-disablecard
+  cp /usr/share/doc/bumblebee/bumblebee-disablecard.template /usr/local/bin/bumblebee-disablecard
  else
   # Already Exists
   echo
-  echo "nVidia card disable-script: /usr/local/bin/bumblebee-disablecard, already exists not overwriting."
+  echo "nVidia card disable-script: /usr/local/bin/bumblebee-disablecard exists, moving old script to /usr/local/bin/bumblebee-disablecard.old."
   echo
+  mv /usr/local/bin/bumblebee-disablecard /usr/local/bin/bumblebee-disablecard.old
+  cp /usr/share/doc/bumblebee/bumblebee-disablecard.template /usr/local/bin/bumblebee-disablecard
  fi
+fi
+fi
+
 chmod +x /usr/local/bin/bumblebee*
 
 update-rc.d bumblebee defaults
